src/io/output.f90

```diff
@@ -76,12 +76,14 @@ module subroutine write_output(output, options)
             call io_write(filename, trim(output%variables(i)%name), output_data, dimnames)
             call io_add_attribute(filename, "description", "predicted mean value", trim(output%variables(i)%name))
             call io_add_attribute(filename, "coordinates", "lon lat time", trim(output%variables(i)%name))
+            call io_add_attribute(filename, "missing_value", real(kFILL_VALUE), trim(output%variables(i)%name))
```

Comment:
I think this should be `_FillValue` instead of `missing_value`.

Reply:
I tried that at first, but the netcdf library won't let you use attributes that start with `_`  I have to make more substantial changes to have it call a different subroutine to define the fill value when the variable is first created... it's not that hard, I was just avoiding it for now.  Will it break any of your post-processing if `_FillValue` or `missing_value` are added?  I'll see if I can just add `_FillValue` properly.

Reply:
fixed